Source/Client/Graphics/Direct3D.cs

```diff
@@ -2006,7 +2006,7 @@ public struct MVertex
 	{
 		// Vertex format
 		public static readonly VertexFormat Format = VertexFormat.Position | VertexFormat.Texture3 | VertexFormat.Diffuse;
-		public static readonly int Stride = 10 * 4;
+		public const int Stride = 10 * 4;
```

Comment:
Should i change it to

```c#
public static readonly int Stride = Unsafe.SizeOf<MVertex>();
```

So if the layout changes, we will get updated stride with no modifications?

Reply:
I'll change it myself.